src/sentry/event_manager.py

```diff
@@ -566,6 +566,28 @@ def save(self, project_id, raw=False, assume_normalized=False):
         if transaction_name:
             transaction_name = force_text(transaction_name)
 
+        # Right now the event type is the signal to skip the group. This
+        # is going to change a lot.
+        if event.get_event_type() == 'transaction':
+            create_group = False
+        else:
+            create_group = True
+
+        org = Organization.objects.get_from_cache(
+            id=project.organization_id
+        )
+        issueless_enabled = features.has('organizations:issueless-events', organization=org)
+        if not create_group and not issueless_enabled:
+            # We discard the event if issueless events are disabled
+            logger.info(
+                'issueless.disabled',
+                extra={
+                    'event_uuid': data['event_id'],
+                    'project_id': project.id,
+                }
+            )
```

Comment:
FYI @JTCunning.

Reply:
This is fine as of now. If anything, it will alert us as to people who are possibly doing things they shouldn't.